fastestimator/summary/system.py

```diff
@@ -335,6 +343,8 @@ def _load_dict(states: Dict[str, Any], state_key: str, in_memory_objects: Dict[A
                 obj.__setstate__(state)
             elif hasattr(obj, '__dict__'):
                 obj.__dict__.update(state)
+            elif isinstance(obj, dict):
+                [self._load_dict(states, k, obj) for k in states]
```

Comment:
this can stay as a static method by invoking System._load_dict

Reply:
static now